reproject/interpolation/tests/test_core.py

```diff
@@ -586,6 +586,25 @@ def test_identity_with_offset(roundtrip_coords):
 @pytest.mark.parametrize('parallel', [True, 0, 2, False])
 @pytest.mark.parametrize('block_size', [[10, 10], [500, 500], [500, 100], None])
 def test_blocked_against_single(parallel, block_size):
+    # the warning import and ignore is needed to keep pytest happy when running with
+    # older versions of astropy which don't have this fix:
+    # https://github.com/astropy/astropy/pull/12844
+    # All the warning code should be removed when old version no longer being used
+    import warnings
+    warnings.simplefilter('ignore', category=FITSFixedWarning)
```

Comment:
Does this ignore it for the whole test session? Would a context manager be safer?

Reply:
I did a quick test by just adding the following function to the bottom of the test file and it still failed correctly - so no doesn't seem to affect the whole pytest session. 
```
def test_fitswarning():
    raise FITSFixedWarning
```

I assume each test gets its own new invocation of the python interepter so it works but couldn't find any conrete reference to that implementation detail or if it should be relied upon.

So I do agree it would be better if a ` with warnings.catch_warnings()` manager was used, since then it could be used just wrap the new changes - I've done this locally and just running some checks